akka-actor/src/main/resources/reference.conf

```diff
@@ -1080,17 +1087,22 @@ akka {
       before-service-unbind {
       }
 
-      # Stop accepting new incoming requests in for example HTTP.
+      # Stop accepting new incoming requests.
```

Comment:
"Stop accepting new incoming connections" would be more real wording here.
Our unbind does not kill existing connections (currently);

It should be implemented in https://github.com/akka/akka-http/issues/1210 as we don't do it nowadays still.

See also  https://github.com/akka/akka-http/issues/575